applications/electron/scripts/update-checksum.ts

```diff
@@ -0,0 +1,104 @@
+/********************************************************************************
+ * Copyright (C) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import * as crypto from 'crypto';
+import * as fs from 'fs';
+import * as jsyaml from 'js-yaml';
+import * as path from 'path';
+import { hideBin } from 'yargs/helpers';
+import yargs from 'yargs/yargs';
+
+
+const argv = yargs(hideBin(process.argv))
+    .option('executable', { alias: 'e', type: 'string', default: '../dist/TheiaBlueprint.AppImage', desription: 'The executable for which the checksum needs to be updated' })
+    .option('yaml', { alias: 'y', type: 'string', default: '../dist/latest-linux.yml', desription: 'The yaml file where the checksum needs to be updated' })
+    .version(false)
+    .wrap(120)
+    .parseSync();
+
+execute();
+
+async function execute(): Promise<void> {
+    const executable = argv.executable;
+    const yaml = argv.yaml;
+
+    const executablePath = path.resolve(
+        __dirname,
```

Comment:
Maybe we could normalize the path, so that it's assumed the executable and yaml file will be under `dist/`? 
e.g.: 
```ts
const executablePath = path.resolve(
   __dirname,
   `../dist`,
   executable
_;
```

Reply:
Sounds good. I've made the change and the build looks fine: https://ci.eclipse.org/theia/job/Theia%20PRs/view/change-requests/job/PR-129/14/artifact/applications/electron/dist/